base/src/main/java/org/mozilla/jss/crypto/PBEKeyGenParams.java

```diff
@@ -31,7 +32,7 @@ public class PBEKeyGenParams implements AlgorithmParameterSpec, KeySpec {
      * @param iterations The iteration count for the PBE algorithm.
      */
     public PBEKeyGenParams(Password pass, byte[] salt, int iterations) {
-        if (pass == null || salt == null) {
+        if (pass == null) {
```

Comment:
So are we now accounting for a case where the salt is null here?

Reply:
Sorry, this is a mistake. I am going to fix it.

Reply:
Looks good.